ext/php5/signals.c

```diff
@@ -1,6 +1,9 @@
 #include "signals.h"
 
-#include "config.h"
+#ifdef HAVE_CONFIG_H
+# include "config.h"
```

Comment:
The linter is [failing in CI](https://app.circleci.com/pipelines/github/DataDog/dd-trace-php/5796/workflows/51e98ae7-81b8-4562-b2f6-dbeb4533fc94/jobs/438549) on the extra spaces in these files. This particular one has gotten me a few times. We use `clang-format` for our linting/formatting.

Reply:
In my experience, `clang-format` always removes all the whitespace around compiler directives.

I believe our options are:

- Add the [`IndentPPDirectives: AfterHash` rule](https://stackoverflow.com/a/48610210/443479) to `.clang-format` to allow/enforce indenting embedded compiler directives. (This will impact the entire project and require a largish diff so I would love to know @morrisonlevi's thoughts here. But I'm 👍 for this option in a separate PR.)
- Flatten it out to conform to the existing linter rules. (I usually just run `clang-format -i filename.c` before pushing up to GH so that I don't have to think about formatting.)
- Use the `// clang-format {off|on}` comments before and after the lines the linter is complaining about. ([example](https://github.com/DataDog/dd-trace-php/blob/f417c10/zend_abstract_interface/zai_sapi/zai_sapi_functions.c#L82-L88))

Reply:
I would be in favour as well, I find properly indented preprocessor directives much more readable.

Reply:
I reformatted it for now, until we reach a decision on the indent.